.gitignore

```diff
@@ -0,0 +1,12 @@
+/build
+
+# Mac
+.DS_Store
+
+#VSCode
+.vscode
+
+# Intellij
+.idea/
+*.iml
+*.iws
```

Comment:
Need a new line.  GitHub it seems prefers this: https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log